src/ios/CDVWKInAppBrowser.m

```diff
@@ -257,16 +257,18 @@ - (void)openInInAppBrowser:(NSURL*)url withOptions:(NSString*)options
     
     [self.inAppBrowserViewController navigateTo:url];
     if (!browserOptions.hidden) {
-        [self show:nil withNoAnimate:browserOptions.hidden];
+        [self show:nil withNoAnimate:browserOptions.hidden displayCustomToolbar:browserOptions.customToolbar toolbarTitle:browserOptions.customToolbarTitle toolbarForegroundColor:[CDVWKInAppBrowserViewController colorFromHexString:browserOptions.customToolbarFg] toolbarBackgroundColor:[CDVWKInAppBrowserViewController colorFromHexString:browserOptions.customToolbarBg]];
     }
 }
 
 - (void)show:(CDVInvokedUrlCommand*)command{
-    [self show:command withNoAnimate:NO];
+    [self show:command withNoAnimate:NO displayCustomToolbar:NO toolbarTitle:nil toolbarForegroundColor:nil toolbarBackgroundColor:nil];
 }
 
-- (void)show:(CDVInvokedUrlCommand*)command withNoAnimate:(BOOL)noAnimate
+- (void)show:(CDVInvokedUrlCommand*)command withNoAnimate:(BOOL)noAnimate displayCustomToolbar:(BOOL)displayCustomToolbar toolbarTitle:(NSString *)toolbarTitle toolbarForegroundColor:(UIColor *)toolbarForegroundColor toolbarBackgroundColor:(UIColor *)toolbarBackgroundColor
```

Comment:
Same comment as above. This line will grow longer as more parameters are added. I suggest breaking it up into multiple lines.